modules/ROOT/pages/6.4.2-release-notes.adoc

```diff
@@ -76,6 +76,13 @@ include::partial$misc/admon-releasenotes-for-stable.adoc[]
 
 // {productname} 6.4.2 also includes the following bug fixes:
 
+=== Redial would, in some circumstances, cause elements to not have an initial value selected when they should have.
+
+In previous versions of {productname}, the host browser did not properly manage switching child elements when the Accessibility Checker dialog was visible.
+
+As a consequence, the dropdown menu in this dialog would, in some circumstances, present without one of the dropdown menu’s options being selected by default.
+
+To fix this issue {productname} now populates the element with a default value, unless this menu is explicitly configured as empty by the particular {productname} instance’s configuration.
```

Comment:
```suggestion
To fix this issue {productname} now populates the element with a default value, unless this menu is explicitly configured as empty by the particular {productname} instance’s configuration.

NOTE: this change also addresses xref:6.4.1-release-notes.adoc#known-issues[a known issue] in the xref:a11ychecker.adoc[Accessibility Checker Premium plugin].
```

Conveniently enough, the Accessibility Checker known issue is the first item in the [‘Known Issues’ section of the _6.4.1 Release Notes_](https://tiny.cloud/docs/tinymce/6/6.4.1-release-notes/#known-issues). So, an xref to the `known-issues` section header links to said known issue, at least for practical purposes.